src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -388,7 +388,7 @@ public String getActionInfo(){
     
     public String getMessageInfo(){
         
-        Sanitizer.sanitizeForHtml(message);
+ 
```

Comment:
After removing the code, make sure to keep to the code guidelines, remove unnecessary blank lines